strawberryfields/ops.py

```diff
@@ -110,10 +110,10 @@ def square(q):
         Dgate(RR(q[0])) | q[1]
         Dgate(RR(q[0], lambda q: q ** 2)) | q[2]
 
-* Meta-operations such as :class:`Delete` and :class:`New_modes` Operations delete
+* Meta-operations such as :class:`Delete` and :class:`New_modes` delete
   and create modes during program execution.
   In practice the user only deals with the pre-constructed
-  instances :py:data:`Del` and :py:data:`New`::
+  instance :py:data:`Del` and the function :func:`New`::
```

Comment:
Since `Del` and `New` are user facing, perhaps they should come first in this sentence, and the meta operations mentioned after?

Reply:
Fixed.